pcdsdevices/pseudopos.py

```diff
@@ -167,4 +168,23 @@ class SimDelayStage(DelayBase):
 
 class PseudoSingleInterface(PseudoSingle, FltMvInterface):
     """PseudoSingle with FltMvInterface mixed in."""
-    pass
+    notepad_setpoint = Cpt(
+        NotepadLinkedSignal, ':OphydSetpoint',
+        notepad_metadata={'record': 'ao', 'default_value': 0.0},
+    )
+
+    notepad_readback = Cpt(
+        NotepadLinkedSignal, ':OphydReadback',
+        notepad_metadata={'record': 'ai', 'default_value': 0.0},
+    )
```

Comment:
The implication here is that all of our PseudoSingles will have PVs associated with them, which I think is good. It does mean that these devices will load poorly (`connected=False`) until we run the notepad IOC though.

Reply:
Definitely - without this IOC in place, things will break.

I'm hoping that the `find` tool will be run periodically either through a cron job or in a spawned subprocess from the IOC itself, making integrating these pretty seamless. (That's where the periodic checks for reloading that file come in, even if it's not really used currently.)

Reply:
Will we need to run an instance of this IOC on every subnet?

Reply:
I hadn't considered breaking it up between subnets, rather assuming it was truly global. 

If we do split it up, we'd want to be selective about which PVs were on which IOC. Can that be done cleanly? Beamline metadata in happi, I assume?

Reply:
If it's global, where do we run the IOC so that all consumers can get their respective data, and so that all python sessions can put to the global PV? Just things to think about.

Some combination of happi metadata and PV naming should be enough to decide where these go if we split it up. Ideally we want to hit the same subnet as the associated real motor PVs.

Reply:
I don't think either way will be perfectly clean but I vote for splitting it up. I think the 'beamline' field in happi should probably be enough to tell you each IOC which objects it needs.

Doing it that way will still lead to lead to some gateway issues for shared objects (off the top of my head, I'm thinking of the CXI/XCS LODCM), but that's easier and/or cleaner than any method that I could imagine for setting up the gateway with this running globally.

Reply:
OK, will add in parameters for splitting up the config in the IOC's `notepad_finder` helper.